### PR TITLE
Add $lib.env.get() to Storm to access env vars which start with SYN_STORM_ENV_

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/tests/test_lib_stormlib_env.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import os
+import synapse.exc as s_exc
+import synapse.tests.utils as s_test
+
+class StormLibEnvTest(s_test.SynTest):
+
+    async def test_stormlib_env(self):
+
+        os.environ['SYN_STORM_ENV_WOOT'] = 'woot'
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_stormlib_env.py:9]: Use this context manager to set/unset these:

```
with self.setTstEnvars(SYN_STORM_ENV_WOOT='woot'):
```

File: synapse/lib/stormlib/env.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+        self.runt.reqAdmin(mesg='$lib.env.get() requires admin privileges.')
+
+        name = await s_stormtypes.tostr(name)
+
+        if not name.startswith('SYN_STORM_ENV_'):
+            mesg = f'Environment variable must start with SYN_STORM_ENV_ : {name}'
+            raise s_exc.BadArg(mesg=mesg)
+
+        return os.getenv(name)
```
Comment (mikemoritz) [inline synapse/lib/stormlib/env.py:41]: Did you consider plumbing the default kwarg?

File: synapse/tests/test_lib_stormlib_env.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+        async with self.getTestCore() as core:
+
+            self.eq('woot', await core.callStorm('return($lib.env.get(SYN_STORM_ENV_WOOT))'))
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_stormlib_env.py:13]: need a test for a non-existent environment variable

File: synapse/lib/stormlib/env.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import synapse.common as s_common
+import synapse.lib.stormtypes as s_stormtypes
+
+@s_stormtypes.registry.registerLib
+class LibEnv(s_stormtypes.Lib):
+    '''
+    A Storm Library for accessing environment vars.
+    '''
+    _storm_locals = (
+        {'name': 'get', 'desc': 'Retrieve an environment variable.',
```
Comment (vEpiphyte) [inline synapse/lib/stormlib/env.py:13]: We should note that the variable names must start with `SYN_STORM_ENV_`

**Review by mikemoritz — DISMISSED** [review]

**Review by MichaelSquires — DISMISSED** [review]

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**